tools/scripts/tx_jsons/staking_params.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
Just wanted to add these into the repo. I'll make use of them in the validator staking documentation.